### DISCUSSION TITLE
投稿画面でマウスカーソルの位置により上部バーの横方向のリサイズが繰り返される（振動する）

### DISCUSSION THREAD

**Discussion by tfukumori** [increments/qiita-discussions#181] [要望・アイデア・不具合]
# 現象

投稿画面でマウスカーソルの位置により上部バーの横方向のリサイズが繰り返される（振動する）

# 環境

Windows 10 Pro 20H2, Microsoft Edge
1920 x 1080

# 再現手順

1. ウインドウ最大化の状態で新規記事編集画面を開く
2. ウインドウを最大化から元のサイズに戻す
3. ウインドウの立てスクロールバーが表示される
4. 上部バーの通知アイコンとユーザーアイコンの間にマウスカーソル移動
5. 描画不正（上部バーの横方向のリサイズが繰り返される（横方向に振動する））

**Comment by kskwtnk** [comment]
@tfukumori 
ご意見ありがとうございます！

こちら、一旦確認してみた限りでは再現ができませんでした…… 🙇 
他のデバイスで再現しないかなど、もう少し調査してみます！

今しばらくお待ちください 🙇

**Reply by kskwtnk** [reply]
元ツイート
https://twitter.com/ht_deko/status/1504228283608489984

**Reply by tfukumori** [reply]
関連ツイートの記載ありがとうございます🙇‍♂️

**Reply by ht-deko** [reply]
1. 記事を新規作成。
2. Chrome のウィンドウを横方向に小さくなるようにリサイズ (スクロールバーが出るまで)。
3. 上部バーのユーザーアイコンの右側か左側にマウスポインタを移動。
4. 振動する。
5. Chrome のウィンドウを横方向または縦方向に大きくなるようにリサイズ (スクロールバーが消えるまで)。
6. 上部バーのユーザーアイコンの右側か左側にマウスポインタを移動。
7. 振動しない。

既存の記事を編集すると、上記手順の 2. が実行された状態と同様になるものがあり、
それらの記事では単にアイコンの右側か左側にマウスポインタを移動しただけで振動します。

**Comment by ohakutsu** [comment]
@tfukumori @ht-deko 
遅くなってしまい、申し訳ございません :bow:

こちら調査をしてみたのですが、完全には再現ができませんでした...
ですが、調査中にご報告いただいている状態に近い状態（横方向のリサイズが発生する状態）は再現ができ、その修正をリリースしました！
もし、まだ問題が発生する様でしたら、ご報告していただけると幸いです :pray:

**Reply by ht-deko** [reply]
こちらでざっと確認した限りでは、記事新規作成時でも、
以前問題の発生していた記事の編集時でも振動したりする事はありませんでした。
修正ありがとうございます！

**Reply by duri0214** [reply]
@ohakutsu 私も編集してみましたがなおってました。感謝します
![image](https://user-images.githubusercontent.com/19668980/161221898-949c04e2-37a8-4387-b589-acc7985299be.png)

**Reply by tfukumori** [reply]
@ohakutsu 改善されていました！（記載した手順で確認）
対応ありがとうございます……！
